python/ray/data/preprocessors/scaler.py

```diff
@@ -99,16 +99,53 @@ def __repr__(self):
 
 
 class MaxAbsScaler(Preprocessor):
-    """Scale values within columns based on the absolute max value.
+    r"""Scale each column by its absolute max value.
```

Comment:
Thinking through this, maybe we want to establish a pattern in the docstrings to describe what fit and transform individually do.